projects/experimental/directives/pin/pin.directive.ts

```diff
@@ -0,0 +1,43 @@
+import {
+    ChangeDetectionStrategy,
+    Component,
+    Directive,
+    EventEmitter,
+    Inject,
+    Input,
+    Output,
+    ViewEncapsulation,
+} from '@angular/core';
+import {TuiDirectiveStylesService} from '@taiga-ui/cdk';
+
+@Component({
+    template: '',
+    styleUrls: ['./pin.styles.less'],
+    encapsulation: ViewEncapsulation.None,
+    changeDetection: ChangeDetectionStrategy.OnPush,
+    host: {
+        class: 'tui-pin',
+    },
+})
+export class TuiPinComponent {}
+
+@Directive({
```

Comment:
I think there's no need for it to be a directive, a component with attribute selector is enough, because it is reliant on its content – it will not mix with other components anyway. But attribute selector would allow us to put it on `button`, for example. BTW, in such cases, what should be our convention, use `tui-pin,[tuiPin]` as selector, or always use `[tuiPin]`, what do you think, @vladimirpotekhin @nsbarsukov. Previously we aloways had 2 selectors, I believe.